Library/Formula/caf.rb

```diff
@@ -0,0 +1,52 @@
+class Caf < Formula
+  homepage "http://actor-framework.org"
+  head "https://github.com/actor-framework/actor-framework.git"
+
+  stable do
+    url "https://github.com/actor-framework/actor-framework/archive/0.12.0.tar.gz"
+    sha256 "294fb00508c166595d479e9af2fae28f2d370ed164cd1de079ae247acc916801"
+  end
+
+  devel do
+    url "https://github.com/actor-framework/actor-framework.git",
+      :branch => "develop"
+    version "0.13-devel"
+  end
+
+  depends_on "cmake" => :build
+
+  option "with-static", "build as static and shared library"
+  option "with-static-only", "build as static library only"
+  option "with-more-warnings", "build with most compiler warnings"
+  option "with-warnings-as-errors", "elevate compiler warnings to errors"
```

Comment:
Unless there's a demand for all these options, we can probably kill some of them.